app/controllers/products_controller.rb

```diff
@@ -6,7 +6,22 @@ def get_vendors
   end
 
   def product_params
-    params.require(:product).permit(:name, :vegan, :gluten_free, :dairy_free, :lc_based, :fair, :eco_sound, :humane, :upc, :vendor_id)
+    params.require(:product).permit(:name, :picture, :vegan, :gluten_free, :dairy_free, :organic, 
```

Comment:
Can we condense these? This is a lot of parameters to just pass in, and potentially problematic down the line if a change is made, especially if more parameters need to be passed in.

Reply:
To be honest I'm not too sure how we should handle this. Perhaps we can have a separate strong params method that just handle tags. However, all the parameters here are necessary, so the only thing we can really do is make it look nicer. We can probably handle this later (in this iteration in a separate branch or in next iteration).